src/main/java/salesforce/ui/pages/legalentity/LegalEntityClassicVersion.java

```diff
@@ -0,0 +1,160 @@
+package salesforce.ui.pages.legalentity;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class LegalEntityClassicVersion extends LegalEntityPageAbstract {
+
+    @FindBy(id = "Name_ileinner")
+    private WebElement name;
+
+    @FindBy(id = "CompanyName_ileinner")
+    private WebElement companyName;
+
+    @FindBy(id = "LegalEntityAddress_ileinner")
+    private WebElement address;
+
+    @FindBy(id = "Description_ileinner")
+    private WebElement description;
+
+    @FindBy(id = "Status_ileinner")
+    private WebElement status;
+
+    @FindBy(xpath = "//h2[@class='pageDescription']")
+    private WebElement headerName;
+
+    private final int time = 2000;
+    private By addressId = By.id("LegalEntityAddress_ileinner");
+
+    /**
+     * Builds a map from the ui.
+     *
+     * @return HashMap<String, String> with the entity map
+     */
+    @Override
+    public HashMap<String, String> entityMap() {
+        HashMap<String, String> entityMap = new HashMap<>();
+        entityMap.put("Name", getNameText());
+        entityMap.put("CompanyName", getCompanyNameText());
+        entityMap.put("LegalEntityStreet", getAddressText().get(0));
+        entityMap.put("Address", getAddressText().get(1));
+        entityMap.put("Country", getAddressText().get(2));
+        entityMap.put("Description", getDescriptionText());
+        entityMap.put("Status", getStatusText());
+        return entityMap;
+    }
+
+    /**
+     * Gets the entity name text.
+     *
+     * @return a string with the entity name text.
+     */
+    @Override
+    public String getHeaderEntityNameText() {
+        return fixText(getWebElementAction().getTextOnWebElement(headerName));
+    }
+
+    /**
+     * There is no message in the lightning skin.
+     *
+     * @return .
```

Comment:
description for return in javadoc is missing

Reply:
👍🏻